app/scripts/modules/amazon/serverGroup/configure/wizard/advancedSettings/advancedSettings.directive.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.amazon.serverGroup.configure.wizard.advancedSettings.directive', [
+  ])
+  .directive('awsServerGroupAdvancedSettings', function () {
+    return {
+      restrict: 'E',
+      templateUrl: require('./advancedSettings.directive.html'),
+      scope: {},
+      bindToController: {
+        command: '=',
+      },
+      controllerAs: 'vm',
+      controller: angular.noop,
+    };
+  });
```

Comment:
The component version of this would look like:

```javascript
.component('awsServerGroupAdvancedSettings', {
    bindings: {
      command: '='
    },
    templateUrl: require('./advancedSettings.component.html') // you would want to rename the template file to this
  });
```
Then, you'd need to replace all the `vm.` references in the template with `$ctrl`.